2024-08-09 17:47:21,229 ERROR Oracle.DataAccess.Client.OracleException ORA-08004: secuencia SEQ_NROTRAM_GRALDEMO.NEXTVAL exceeds MAXVALUE y no se puede instanciar    en Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, String procedure, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteReader(Boolean requery, Boolean fillRequest, CommandBehavior behavior)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, MergeOption mergeOption, Object[] parameters)
   en System.Data.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQueryAsIEnumerable[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery(Type elementType, String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalSqlNonSetQuery.GetEnumerator()
   en System.Data.Entity.Internal.InternalSqlQuery`1.GetEnumerator()
   en System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   en Repositorio.TramiteRepositorio.GenerarNroTramite() en d:\IGJMET\Repositorio\TramiteRepositorio.cs:línea 109
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 103
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 17:49:47,620 ERROR Oracle.DataAccess.Client.OracleException ORA-08004: secuencia SEQ_NROTRAM_GRALDEMO.NEXTVAL exceeds MAXVALUE y no se puede instanciar    en Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, String procedure, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteReader(Boolean requery, Boolean fillRequest, CommandBehavior behavior)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, MergeOption mergeOption, Object[] parameters)
   en System.Data.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQueryAsIEnumerable[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery(Type elementType, String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalSqlNonSetQuery.GetEnumerator()
   en System.Data.Entity.Internal.InternalSqlQuery`1.GetEnumerator()
   en System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   en Repositorio.TramiteRepositorio.GenerarNroTramite() en d:\IGJMET\Repositorio\TramiteRepositorio.cs:línea 109
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 103
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 17:50:31,175 ERROR Oracle.DataAccess.Client.OracleException ORA-08004: secuencia SEQ_NROTRAM_GRALDEMO.NEXTVAL exceeds MAXVALUE y no se puede instanciar    en Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, String procedure, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteReader(Boolean requery, Boolean fillRequest, CommandBehavior behavior)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, MergeOption mergeOption, Object[] parameters)
   en System.Data.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQueryAsIEnumerable[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery(Type elementType, String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalSqlNonSetQuery.GetEnumerator()
   en System.Data.Entity.Internal.InternalSqlQuery`1.GetEnumerator()
   en System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   en Repositorio.TramiteRepositorio.GenerarNroTramite() en d:\IGJMET\Repositorio\TramiteRepositorio.cs:línea 109
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 103
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 17:59:07,904 ERROR Oracle.DataAccess.Client.OracleException ORA-08004: secuencia SEQ_NROTRAM_GRALDEMO.NEXTVAL exceeds MAXVALUE y no se puede instanciar    en Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, String procedure, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteReader(Boolean requery, Boolean fillRequest, CommandBehavior behavior)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, MergeOption mergeOption, Object[] parameters)
   en System.Data.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQueryAsIEnumerable[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery(Type elementType, String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalSqlNonSetQuery.GetEnumerator()
   en System.Data.Entity.Internal.InternalSqlQuery`1.GetEnumerator()
   en System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   en Repositorio.TramiteRepositorio.GenerarNroTramite() en d:\IGJMET\Repositorio\TramiteRepositorio.cs:línea 109
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 103
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:02:27,830 ERROR Oracle.DataAccess.Client.OracleException ORA-08004: secuencia SEQ_NROTRAM_GRALDEMO.NEXTVAL exceeds MAXVALUE y no se puede instanciar    en Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, String procedure, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteReader(Boolean requery, Boolean fillRequest, CommandBehavior behavior)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, MergeOption mergeOption, Object[] parameters)
   en System.Data.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQueryAsIEnumerable[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery(Type elementType, String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalSqlNonSetQuery.GetEnumerator()
   en System.Data.Entity.Internal.InternalSqlQuery`1.GetEnumerator()
   en System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   en Repositorio.TramiteRepositorio.GenerarNroTramite() en d:\IGJMET\Repositorio\TramiteRepositorio.cs:línea 109
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 103
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:07:53,355 ERROR Oracle.DataAccess.Client.OracleException ORA-08004: secuencia SEQ_NROTRAM_GRALDEMO.NEXTVAL exceeds MAXVALUE y no se puede instanciar    en Oracle.DataAccess.Client.OracleException.HandleErrorHelper(Int32 errCode, OracleConnection conn, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, String procedure, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleException.HandleError(Int32 errCode, OracleConnection conn, String procedure, IntPtr opsErrCtx, OpoSqlValCtx* pOpoSqlValCtx, Object src, Boolean bCheck)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteReader(Boolean requery, Boolean fillRequest, CommandBehavior behavior)
   en Oracle.DataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, MergeOption mergeOption, Object[] parameters)
   en System.Data.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQueryAsIEnumerable[TElement](String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalContext.ExecuteSqlQuery(Type elementType, String sql, Object[] parameters)
   en System.Data.Entity.Internal.InternalSqlNonSetQuery.GetEnumerator()
   en System.Data.Entity.Internal.InternalSqlQuery`1.GetEnumerator()
   en System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   en Repositorio.TramiteRepositorio.GenerarNroTramite() en d:\IGJMET\Repositorio\TramiteRepositorio.cs:línea 109
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 103
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:33:38,777 ERROR System.ServiceModel.Security.MessageSecurityException: Se prohibió la solicitud HTTP con el esquema de autenticación de cliente 'Anonymous'. ---> System.Net.WebException: Error en el servidor remoto: (403) Prohibido.
   en System.Net.HttpWebRequest.GetResponse()
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   --- Fin del seguimiento de la pila de la excepción interna ---

Server stack trace: 
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateAuthentication(HttpWebRequest request, HttpWebResponse response, WebException responseException, HttpChannelFactory`1 factory)
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateRequestReplyResponse(HttpWebRequest request, HttpWebResponse response, HttpChannelFactory`1 factory, WebException responseException, ChannelBinding channelBinding)
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   en System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   en System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   en System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   en System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   en Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request)
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 123
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.ConsultaTimbrado(String respuesta) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 130
   en System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 59
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:33:46,004 ERROR Domain.Exceptions.ServiceEnteComunicationException: Error de comunicacion con el servicio ENTE de cooperación ---> System.ServiceModel.Security.MessageSecurityException: Se prohibió la solicitud HTTP con el esquema de autenticación de cliente 'Anonymous'. ---> System.Net.WebException: Error en el servidor remoto: (403) Prohibido.
   en System.Net.HttpWebRequest.GetResponse()
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   --- Fin del seguimiento de la pila de la excepción interna ---

Server stack trace: 
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateAuthentication(HttpWebRequest request, HttpWebResponse response, WebException responseException, HttpChannelFactory`1 factory)
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateRequestReplyResponse(HttpWebRequest request, HttpWebResponse response, HttpChannelFactory`1 factory, WebException responseException, ChannelBinding channelBinding)
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   en System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   en System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   en System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   en System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   en Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request)
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 123
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.ConsultaTimbrado(String respuesta) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 130
   en System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 59
   --- Fin del seguimiento de la pila de la excepción interna ---
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 74
   en Gateway.ENTEServiceClient.ConsultarTimbrado(Formulario formulario, Int64 nroTramite, Usuario usuario) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 43
   en Domain.Logics.TramiteLogic.IniciarTransaccion(Formulario formulario, Int32 nroTramite, Operacion operacion, PresentacionTramite presentacion, Usuario usuario, String codTramite) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 349
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:33:51,271 ERROR Domain.Exceptions.ServiceEnteComunicationException: Error de comunicacion con el servicio ENTE de cooperación ---> System.ServiceModel.Security.MessageSecurityException: Se prohibió la solicitud HTTP con el esquema de autenticación de cliente 'Anonymous'. ---> System.Net.WebException: Error en el servidor remoto: (403) Prohibido.
   en System.Net.HttpWebRequest.GetResponse()
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   --- Fin del seguimiento de la pila de la excepción interna ---

Server stack trace: 
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateAuthentication(HttpWebRequest request, HttpWebResponse response, WebException responseException, HttpChannelFactory`1 factory)
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateRequestReplyResponse(HttpWebRequest request, HttpWebResponse response, HttpChannelFactory`1 factory, WebException responseException, ChannelBinding channelBinding)
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   en System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   en System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   en System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   en System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   en Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request)
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 123
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.ConsultaTimbrado(String respuesta) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 130
   en System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 59
   --- Fin del seguimiento de la pila de la excepción interna ---
   en Domain.Logics.TramiteLogic.IniciarTransaccion(Formulario formulario, Int32 nroTramite, Operacion operacion, PresentacionTramite presentacion, Usuario usuario, String codTramite) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 396
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 111
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:37:28,471 ERROR System.ServiceModel.Security.MessageSecurityException: Se prohibió la solicitud HTTP con el esquema de autenticación de cliente 'Anonymous'. ---> System.Net.WebException: Error en el servidor remoto: (403) Prohibido.
   en System.Net.HttpWebRequest.GetResponse()
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   --- Fin del seguimiento de la pila de la excepción interna ---

Server stack trace: 
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateAuthentication(HttpWebRequest request, HttpWebResponse response, WebException responseException, HttpChannelFactory`1 factory)
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateRequestReplyResponse(HttpWebRequest request, HttpWebResponse response, HttpChannelFactory`1 factory, WebException responseException, ChannelBinding channelBinding)
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   en System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   en System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   en System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   en System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   en Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request)
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 123
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.ConsultaTimbrado(String respuesta) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 130
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 59
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:37:50,300 ERROR Domain.Exceptions.ServiceEnteComunicationException: Error de comunicacion con el servicio ENTE de cooperación ---> System.ServiceModel.Security.MessageSecurityException: Se prohibió la solicitud HTTP con el esquema de autenticación de cliente 'Anonymous'. ---> System.Net.WebException: Error en el servidor remoto: (403) Prohibido.
   en System.Net.HttpWebRequest.GetResponse()
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   --- Fin del seguimiento de la pila de la excepción interna ---

Server stack trace: 
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateAuthentication(HttpWebRequest request, HttpWebResponse response, WebException responseException, HttpChannelFactory`1 factory)
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateRequestReplyResponse(HttpWebRequest request, HttpWebResponse response, HttpChannelFactory`1 factory, WebException responseException, ChannelBinding channelBinding)
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   en System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   en System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   en System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   en System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   en Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request)
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 123
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.ConsultaTimbrado(String respuesta) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 130
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 59
   --- Fin del seguimiento de la pila de la excepción interna ---
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 74
   en Gateway.ENTEServiceClient.ConsultarTimbrado(Formulario formulario, Int64 nroTramite, Usuario usuario) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 43
   en Domain.Logics.TramiteLogic.IniciarTransaccion(Formulario formulario, Int32 nroTramite, Operacion operacion, PresentacionTramite presentacion, Usuario usuario, String codTramite) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 349
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack2024-08-09 18:37:55,779 ERROR Domain.Exceptions.ServiceEnteComunicationException: Error de comunicacion con el servicio ENTE de cooperación ---> System.ServiceModel.Security.MessageSecurityException: Se prohibió la solicitud HTTP con el esquema de autenticación de cliente 'Anonymous'. ---> System.Net.WebException: Error en el servidor remoto: (403) Prohibido.
   en System.Net.HttpWebRequest.GetResponse()
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   --- Fin del seguimiento de la pila de la excepción interna ---

Server stack trace: 
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateAuthentication(HttpWebRequest request, HttpWebResponse response, WebException responseException, HttpChannelFactory`1 factory)
   en System.ServiceModel.Channels.HttpChannelUtilities.ValidateRequestReplyResponse(HttpWebRequest request, HttpWebResponse response, HttpChannelFactory`1 factory, WebException responseException, ChannelBinding channelBinding)
   en System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   en System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   en System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   en System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   en System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   en System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   en Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request)
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.Gateway.ENTE.WS_RespuestaEnteSoap.ConsultaTimbrado(ConsultaTimbradoRequest request) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 123
   en Gateway.ENTE.WS_RespuestaEnteSoapClient.ConsultaTimbrado(String respuesta) en d:\IGJMET\Gateway\Service References\ENTE\Reference.cs:línea 130
   en Gateway.ENTEServiceClient.ConsumirServicio(Object proxy) en d:\IGJMET\Gateway\ENTEServiceClient.cs:línea 59
   --- Fin del seguimiento de la pila de la excepción interna ---
   en Domain.Logics.TramiteLogic.IniciarTransaccion(Formulario formulario, Int32 nroTramite, Operacion operacion, PresentacionTramite presentacion, Usuario usuario, String codTramite) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 396
   en Domain.Logics.TramiteLogic.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Dominio\Logics\TramiteLogic.cs:línea 111
   en Services.TramiteService.IniciarTramitePago(String codFormulario, String operador) en d:\IGJMET\Service\TramiteService.cs:línea 77
 Utils.Logger d:\IGJMET\Utils\Logger.cs 42 Error Utils.Logger.Error(d:\IGJMET\Utils\Logger.cs:42) Utils.Logger stack